projects/packages/classic-theme-helper/src/class-main.php

```diff
@@ -39,7 +40,8 @@ public static function init() {
 			self::$instance = new Main();
 			self::$instance->load_modules();
 			// TODO Commenting below since we still load them from theme-tools module
-			// add_action( 'init', array( __CLASS__, 'jetpack_load_theme_tools' ), 30 );
+			add_action( 'init', array( __CLASS__, 'jetpack_load_theme_tools' ), 30 );
```

Comment:
The `Jetpack_Color` and `Tonesque` classes loaded from the theme-tools module have existence guards so this is  safe.